chart/kube-arangodb/values.yaml

```diff
@@ -34,6 +34,7 @@ operator:
     k8sToK8sClusterSync: false
     ml: false
     analytics: false
+    networking: true
```

Comment:
why this is enabled by default?

Reply:
As designed - will be required for core functionality